setup.cfg

```diff
@@ -1,7 +1,3 @@
-[options]
-packages =
-    unit_scaling
-
```

Comment:
I guess most of this file could go into pyproject.toml, but perhaps still not flake8.

That said, unless you can delete this file entirely, I don't see much benefit in moving.

Reply:
Yeah, that was my thinking entirely - looking at Paul's stuff it seemed like he could move [everything except flake8](https://github.com/graphcore-research/jax-scalify/blob/main/setup.cfg), which made me not bother. Though [switching to better tools](https://github.com/thecharlieblake/lovely-llama/blob/main/pyproject.toml) should fix this 🙃